RELEASE_NOTES.md

```diff
@@ -28,6 +42,18 @@ v1.0.0-alpha.x
   [#743](https://github.com/OpenAssetIO/OpenAssetIO/issues/743)
 
 
+### New features
+
+- Added entry point based discovery of Python manager plugins. This
+  allows pure Python manager plugins to be deployed and managed using
+  `pip`, without the need to wrangle `OPENASSETIO_PLUGIN_PATH`. The
```

Comment:
It is mainly pip yes (since pip is the main installer), but not only. The entry point mechanism isn't specific to pip. So maybe "deployed and managed as Python packages"?

Reply:
Good idea, that's much better - updated [here](https://github.com/OpenAssetIO/OpenAssetIO/pull/768/commits/0d2f50c200a87e35ab38b09d121aa8c8ee48d1ff).